Xplat/src/main/java/vazkii/botania/common/item/ManufactoryHaloItem.java

```diff
@@ -38,8 +54,51 @@ public void inventoryTick(ItemStack stack, Level world, Entity entity, int pos,
 	}
 
 	@Override
-	public ResourceLocation getGlowResource() {
-		return glowTexture;
+	public ResourceLocation getGlowResource(ItemStack stack) {
+		return isPassive(stack) ? super.getGlowResource(stack) : glowTexture;
+	}
+
+	@Override
+	public void appendHoverText(ItemStack stack, Level world, List<Component> stacks, TooltipFlag flags) {
+		if (isPassive(stack)) {
+			stacks.add(Component.translatable("botaniamisc.inactive"));
+		} else {
+			stacks.add(Component.translatable("botaniamisc.active"));
+		}
 	}
 
+	@NotNull
+	@Override
+	public InteractionResultHolder<ItemStack> use(Level world, Player player, @NotNull InteractionHand hand) {
+		ItemStack stack = player.getItemInHand(hand);
+		if (getSegmentLookedAt(stack, player) == 0 && player.isSecondaryUseActive()) {
+			togglePassive(stack, player, world);
+			return InteractionResultHolder.sidedSuccess(stack, world.isClientSide());
+		}
+
+		return super.use(world, player, hand);
+	}
+
+	@Override
+	public boolean overrideOtherStackedOnMe(ItemStack stack, ItemStack cursor, Slot slot, ClickAction click,
+			Player player, SlotAccess access) {
+		Level world = player.getLevel();
+		if (click == ClickAction.SECONDARY && slot.allowModification(player) && cursor.isEmpty()) {
+			togglePassive(stack, player, world);
+			access.set(cursor);
+			return true;
+		}
+		return false;
+	}
+
+	private void togglePassive(ItemStack stack, LivingEntity living, Level world) {
+		ItemNBTHelper.setBoolean(stack, TAG_PASSIVE, !isPassive(stack));
+		if (living instanceof Player player && world != null) {
+			world.playSound(player, player.getX(), player.getY(), player.getZ(), BotaniaSounds.manufactoryHaloConfigure, SoundSource.NEUTRAL, 1F, 1F);
+		}
+	}
+
+	private static boolean isPassive(ItemStack stack) {
+		return ItemNBTHelper.getBoolean(stack, TAG_PASSIVE, false);
```

Comment:
why is this stored as passive instead of active? Usually you'd store active instead so reading the code was confusing at first glance.

If it's a forward-compatibility concern that old worlds won't have the active tag, just make it default to `true` instead of `false`.